### ISSUE TITLE
Transfer Hooks

### ISSUE THREAD

**Issue by michaelbryan92** [raydium-io/raydium-clmm#102]
I was trying to deploy a CLMM with a token using the transfer hook token extension. The token extension is disabled, yet I still cannot make a pool. 

I see PYUSD has made a CLMM pool with the same configuration. Can anyone assist e in replicating a CLMM like PYUSD/SOL

**Comment by magentoman2009** [comment]
Hi @michaelbryan92 

Did you ever figure this out? I'm having the same issue. Solana recommends hooks to help token developers like myself add extra protection for tokens, but Raydium won't let me create the pool either.

**Comment by magj2006** [comment]
I guess 

> I was trying to deploy a CLMM with a token using the transfer hook token extension. The token extension is disabled, yet I still cannot make a pool.
> 
> I see PYUSD has made a CLMM pool with the same configuration. Can anyone assist e in replicating a CLMM like PYUSD/SOL

I guess they added PYUSD into whitelist

**Comment by RainRaydium** [comment]
Yes, the `transfer hook` extension is not supported yet, PYUSD is handled in the whitelist.
